volumes/miovision/sql/function-aggregate-volumes_15min.sql

```diff
@@ -0,0 +1,62 @@
+﻿-- Function: miovision.aggregate_15_min()
```

Comment:
For some reason these files show as different names across the two branches on Github